src/screens/Home/HomeScreen.tsx

```diff
@@ -26,6 +31,13 @@ const HomeScreen = ({ navigation }: any) => {
     <PageContainer>
       <TextRegular>Welcome {name}!</TextRegular>
       <TextRegular>Your UID is: {uid}</TextRegular>
+      {Object.keys(cases).map((id: any) => (
```

Comment:
[NO ACTION NECESSARY] Very clean!! remember to include key={id} because React expects a unique key for every list item. [See short explanation here ](https://sentry.io/answers/unique-key-prop/)